addons/editor-devtools/DevTools.js

```diff
@@ -484,12 +249,6 @@ export default class DevTools {
     return ids;
   }
 
-  /**
-   * Initiates a drag event for all block stacks except those in the set of ids.
-   * But why? - Because we know all the ids of the existing stacks before we paste / duplicate - so we can find the
-   * new stack by excluding all the known ones.
-   * @param ids Set of previously known ids
-   */
```

Comment:
Was this comment intentionally removed? It seems useful to me. Same for a few other comments in the same file.

Reply:
@Joeclinton1